tools/pod_translator.py

```diff
@@ -0,0 +1,156 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+import os
+import json
+import sys
+from unittest import TestCase
+from zipfile import ZipFile
+from rich.console import Console
```

Comment:
see if you can avoid using rich (to avoid having dependencies). If needed, make sure you note that this is a dependency (you can make a long comment at the top after the header)